internal/docs/design.md

```diff
@@ -142,15 +142,15 @@ The naming convention is `<provider><product><api>`, where:
     *   Include for local/test packages like (e.g., `fileblob`, `mempubsub`).
     *   May be omitted when it makes the package name too long (e.g. `awssnssqs`
         is long enough, don't add `pubsub`).
-    *   Encouraged when it helps distinguish the package from the provider's own
+    *   Encouraged when it helps distinguish the package from the service's own
         package name (e.g., `s3blob` not `s3`).
 
 ## Portable Type Constructors
 
-Portable type constructors are the functions defined in provider-specific
-packages that end users call to get an instance of the portable type. For
-example, `gcsblob.OpenBucket`, which returns an instance of the `*blob.Bucket`
-portable type backed by GCS.
+Portable type constructors are the functions defined in driver packages that end
```

Comment:
service-specific driver packages?

Reply:
I think driver packages is fine here. It's shorter, and some drivers are not necessarily service-specific (for example, the Mongo driver is usable against multiple services).